### ISSUE TITLE
university building icon in building quest nearly invisible in the night mode

### ISSUE THREAD

**Issue by matkoniecz** [westnordost/StreetComplete#1659]
It is in a "civic building" category and as part of icon for dormitory in "residential" category.

**Use case**
Visible icon would be more useful.

**Proposed Solution**
I think that I remember blue variant of university graduation hats? It should be more visible as in icon and still be OK.

**Comment by thymaro** [comment]
I deactivated night mode completely, because the text is black on dark grey. Haven't opened a ticket, since I figured the devs would notice themselves soon enough. Maybe not, though.

**Comment by matkoniecz** [comment]
> Haven't opened a ticket, since I figured the devs would notice themselves soon enough. Maybe not, though.

In my case: I deeply dislike night themes and never use them. Friend complained about some issue triggered in the night theme, so I enabled it and withing 30 seconds I spotted the first broken graphics - now fixed by #1663 (and some other including this issue)

In general, I would not assume that authors use app in the same way, and in general reporting bugs is a good idea (except projects that have already 1000+ reports or are dead, neither apply here). Note for example #1634 that revealed that night mode was completely broken for right-to-left languages. It is something that I would never notice as I use neither night mode nor RTL interface.

If you are not using RTL language then reporting this issue seems to be a good idea.
